docs: :memo: atualizando o README.md

JonasBorgesLM opened the pull request:
Atualizando o o README.md e criacao de um script para automatizar a criacao das tabelas dos dbs

Step 3/4 - Readme
Step 4/4 - Rebasing your branch

JonasBorgesLM committed:
docs: :memo: atualizando o README.md

Atualizando o o README.md e criacao de um script para automatizar a criacao das tabelas dos dbs

matheusflds commented on .env_example:
```diff
@@ -12,3 +12,4 @@ DB_TEST_PORT=5433
 DB_TEST_USERNAME=postgres
 DB_TEST_PASSWORD=postgres
 DB_TEST_DATABASE=userdb
+DB_TEST_URL="postgresql://${DB_TEST_USERNAME}:${DB_TEST_PASSWORD}@${DB_TEST_HOST}:${DB_TEST_PORT}/${DB_TEST_DATABASE}"
```
acabei comentando sobre o env no outro PR fechado ja, acho que dá aproveitar esse PR do readme e ajustar esse ponto do env (antes de vc precisar adicionar mais envs e ter mais retrabalho)

acho que dá pra aproveitar pra ver os pontos do readme tb (nao precisa da doc do graphql no readme)

matheusflds commented on README.md:
````diff
@@ -28,6 +31,8 @@ Este projeto é um sistema de gerenciamento de usuários que permite **criar, le
    ```sh
    git clone https://github.com/indigotech/onboard-jonas-borges.git
    cd onboard-jonas-borges
+   docker-compose up -d
````
```suggestion
   docker compose up -d
```
esse `docker-compose` já não tem suporte em boa parte dos ambientes unix (linux e mac)
Nas actions do github mesmo já não tem

matheusflds commented on package.json:
```diff
@@ -9,7 +9,8 @@
     "format": "prettier --write 'src/**/*.ts'",
-    "lint": "eslint 'src/**/*.{js,ts}'"
+    "lint": "eslint 'src/**/*.{js,ts}'",
+    "migrate:deploy": "npx prisma migrate deploy"
```
```suggestion
    "migrate:deploy": "npm prisma migrate deploy"
```
para garantir que ta usando o que ta instalado no node_modules e nao instalar global sem necessidade

JonasBorgesLM committed:
refactor: :recycle: melhorias levantadas CR

Melhorias apontadas no CR.  Modificacao do .env, documentacao para o uso do docker compose e script de criacao das tabelas

JonasBorgesLM committed:
refactor: :recycle: Adicao da quebra de linha

JonasBorgesLM committed:
refactor: :recycle: melhorias levantadas .env, test.env e docker
